projects/openexr/openexr_deepscanlines_fuzzer.cc

```diff
@@ -151,35 +155,19 @@ static void readFileMulti(const char filename[]) {
 
 }  // namespace
 
-// from cl/164883104
-static char *buf_to_file(const char *buf, size_t size) {
-  char *name = strdup("/dev/shm/fuzz-XXXXXX");
-  int fd = mkstemp(name);
-  if (fd < 0) {
-    perror("open");
-    exit(1);
-  }
-  size_t pos = 0;
-  while (pos < size) {
-    int nbytes = write(fd, &buf[pos], size - pos);
-    if (nbytes <= 0) {
-      perror("write");
-      exit(1);
-    }
-    pos += nbytes;
-  }
-  if (close(fd) != 0) {
-    perror("close");
-    exit(1);
-  }
-  return name;
-}
-
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
-  char *file = buf_to_file((const char *)data, size);
-  readFileSingle(file);
-  readFileMulti(file);
-  unlink(file);
-  free(file);
+  if (size < 3) return 0;
+
+  FuzzedDataProvider stream(data, size);
+  uint8_t width = stream.ConsumeIntegral<uint8_t>();
```

Comment:
255 is too small of limit for width or height, why uint32_t or uint64_t

Reply:
Use types similar to what is used in source code, that makes us use max allowed limits.